bin/docker-command.bash

```diff
@@ -12,8 +12,14 @@ if [ ! -z "${INSTALL_REQS}" ]; then
 fi
 
 # Provision the Django test environment.
-python manage.py createcachetable
-python manage.py collectstatic --noinput -i other &
-python manage.py migrate
-python manage.py get_prices
-python manage.py runserver 0.0.0.0:8000
+CONTAINER_NAME=$(cat /proc/self/cgroup | grep -o  -e "docker-.*.scope" | head -n 1 | sed "s/docker-\(.*\).scope/\\1/")
+if [ "$CONTAINER_NAME" == "shell" ]; then
```

Comment:
@mbeacom this doesnt seem to be a working way of figuring out which container is which from inside the container.. any way you know we can do this?

Reply:
@owocki I don't mean to step on your toes, but I raised another PR that could address this issue at #863 to address the issue without the additional container. Previously, I didn't realize you could simply attach and utilize the ipdb shell.

Reply:
yay!  your approach is cleaner.  let me close this PR and go ahead and merge yours :)